src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -24,7 +25,7 @@ const VideoCard = (props) => {
             <VideoImgWrapper>
                 <Img src={thumbnail} />
             </VideoImgWrapper>
-            <VideoTitle>{escapeString(title)}</VideoTitle>
+            <VideoTitle>{escapeString(title)}<br/><VideoDate>({formatDate(publishedAt)})</VideoDate></VideoTitle>
```

Comment:
I'd use something else instead of the `<br>` to define the layout, a container styled with CSS in more flexible than a `<br>` have the space or the like break, this is the markup that it generates:
```css
<p class="sc-hKgILt gTLZXx">
  The future we're building -- and boring | Elon Musk
  <br>
  <span class="sc-eCssSg hmocIu">(May 03, 2017)</span>
</p>
```
sometimes when the title is big it overflows like in this example
<img width="411" alt="Screen Shot 2021-03-24 at 14 10 45" src="https://user-images.githubusercontent.com/7538600/112376934-bd1c2780-8caa-11eb-9dee-cf47ff146e6c.png">
I'd use a different way to define the layout, not a big deal though